internal/cephfs/volumeoptions.go

```diff
@@ -122,13 +122,13 @@ func getClusterInformation(options map[string]string) (*util.ClusterInfo, error)
 		return nil, err
 	}
 
-	monitors, err := util.Mons(csiConfigFile, clusterID)
+	monitors, err := util.Mons(util.CsiConfigFile, clusterID)
```

Comment:
this should be part of the previous commit where you moved `csiConfigFile` to the `utils` package.

Reply:
done